test/unittests/mocks/Dbus_interface_mock.h

```diff
@@ -1,17 +1,15 @@
-#ifndef SDBUS_C_DBUS_INTERFACE_MOCK_H
-#define SDBUS_C_DBUS_INTERFACE_MOCK_H
+#ifndef SDBUS_CXX_DBUS_INTERFACE_MOCK_H
+#define SDBUS_CXX_DBUS_INTERFACE_MOCK_H
 
-#include <gmock/gmock.h>
+#include "ISdBus.h"
 
-#include "IDbus.h"
+#include <gmock/gmock.h>
 
-class Dbus_interface_mock : public IDbus
+class Dbus_interface_mock : public ISdBus
```

Comment:
For naming consistency, please rename to camel case, e.g. SdBusInterfaceMock or even better SdBusMock (name of production implementation class + "Mock" appended)